test/scripts/integration-tests.sh

```
@@ -0,0 +1,304 @@
+# Function to start the moclojer server
+start_server() {
+    echo "🚀 Starting moclojer server with config: $CONFIG_PATH" >&2
+
+    # Check if config file exists
+    if [ ! -f "$CONFIG_PATH" ]; then
+        echo "❌ Configuration file not found: $CONFIG_PATH" >&2
+        exit 1
+    fi
+
+    # Check if port is available
+    if lsof -i:$SERVER_PORT >/dev/null 2>&1; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)